views/src/Pages/Mentordashboard.jsx

```diff
@@ -1,33 +1,56 @@
-import React from 'react';
-import Sidebar from '../components/Sidebar';
-import { Routes, Route, useLocation } from 'react-router-dom'; 
-import { IconListDetails, IconCheckupList, IconUser } from '@tabler/icons-react';
-import FeedbackRequestQueue from '../Pages/FeedbackQueue';
-import AssignedFeedback from '../Pages/AssignedFeedback';
-import Account from '../Pages/Account';
-import TopBar from '../components/TopBar';
+import React from "react";
+import Sidebar from "../components/Sidebar";
+import { Routes, Route, useLocation } from "react-router-dom";
+import {
+  IconListDetails,
+  IconCheckupList,
+  IconUser,
+  IconTable,
+} from "@tabler/icons-react";
+import FeedbackRequestQueue from "../Pages/FeedbackQueue";
+import AssignedFeedback from "../Pages/AssignedFeedback";
+import Account from "../Pages/Account";
+import TopBar from "../components/TopBar";
+import Checklist from "./Checklist";
 
 function Mentordashboard(props) {
   const navItems = [
-    { icon: IconListDetails, label: 'Feedback Queue', to: 'feedbackqueue' },
-    { icon: IconCheckupList, label: 'Assigned Feedback', to: 'assigned' },
-    { icon: IconUser, label: 'Account', to: 'account' }, 
+    { icon: IconListDetails, label: "Feedback Queue", to: "feedbackqueue" },
+    { icon: IconCheckupList, label: "Assigned Feedback", to: "assigned" },
+    { icon: IconUser, label: "Account", to: "account" },
+    { icon: IconTable, label: "Checklist", to: "checklist" },
   ];
 
-  const location = useLocation(); 
+  const location = useLocation();
 
   return (
-    <div style={{ display: "flex", flexDirection: "column-reverse", position: "relative" }}>
+    <div
```

Comment:
You seem to be using a `div` element here let's maybe change that to a mantine component called `Container`. It's a bit more responsive and fits in with the theme.

Reply:
Alright let me try to work that.